spec/miq_ae_expression_method_spec.rb

```diff
@@ -0,0 +1,169 @@
+module MiqAeExpressionMethodSpec
+  include MiqAeEngine
+  describe MiqAeExpressionMethod do
```

Comment:
Can we come up with a way to test this class without having to build an automate model and instantiate it each time?  Seems very expensive to run each test and it would be better to focus the tests on the class.  Thoughts?